packages/ui/src/components/ui/attachment/useAttachment.ts

```
@@ -0,0 +1,124 @@
+  }, [allowedTypes]);
+
+  const handleRemove = (name: string) => {
+    setFiles((prev) => prev.filter((file) => file.name !== name));
+  };
+
+  useEffect(() => {
+    if (fileUploadMutation.data?.data?.ids?.length) {
+      onChange(fileUploadMutation.data?.data.ids);
+      setFiles((prev) =>
+        prev.map((file) => {
+          delete file.loading;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)